SoftwareConstructionFundamental/SCF-Session8/JUnit.java

```diff
@@ -0,0 +1,53 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.Before;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+
+import zoo.Zoo;
+
+class JUnit {
+	
+	@Test
+    public void addZoneTest() {
+        // Adding a new Zone for Mammals which has Canteen and Park both
+        assertTrue(new Zoo().addZone(2, "Mammal", true, true));
```

Comment:
@meta-shubham-jain , Move to a variable.

Reply:
Done